jbehave-core/src/main/java/org/jbehave/core/parsers/RegexStoryParser.java

```diff
@@ -46,7 +46,12 @@ public RegexStoryParser(Configuration configuration) {
                 configuration.parameterConverters(), configuration.parameterControls());
     }
 
-    public Story parseStory(String storyAsText) {
+    public RegexStoryParser(Configuration configuration, ExamplesTableFactory examplesTableFactory) {
```

Comment:
This approach won't work for cases when step uses parameters of ExamplesTable type